cli/tools/doc.rs

```diff
@@ -98,7 +98,14 @@ pub async fn print_docs(
       get_types(flags.unstable).as_str(),
     )
   } else {
-    let module_specifier = resolve_url_or_path(&source_file).unwrap();
+    let module_specifier = resolve_url_or_path(&source_file);
+    let module_specifier = match module_specifier {
+      Ok(s) => s,
+      Err(e) => {
+        eprintln!("{}", e);
+        std::process::exit(1);
+      }
+    };
```

Comment:
This error should be returned to the caller instead of calling `process::exit` - use: `resolve_url_or_path(&source_file)?` instead

Reply:
fixed